test/openshift/cluster.test.ts

```diff
@@ -17,6 +17,7 @@ import { TestItem } from './testOSItem';
 import { OpenShiftItem } from '../../src/openshift/openshiftItem';
```

Comment:
This does not look like a test for a change. It looks like do stubbing to make test to pass. Methods from Filters class should work just fine if test data is correct. So I would stick to fixing test data to make tests pass. Now it fails because of changes in error message generation. 

```
const err = new Error('FATAL ERROR');
err['cmd'] = 'command text'; 
```

instead of just text.